build.gradle

```diff
@@ -167,22 +167,27 @@ eclipseClasspath {
     }
 }
 
```

Comment:
If somebody accidentally deleted the MOTD placeholder (like what happened recently), would that be detected after this PR is applied?